test/kokoro/xds_k8s_lb.sh

```diff
@@ -167,6 +167,7 @@ main() {
     "outlier_detection_test"
     "remove_neg_test"
     "round_robin_test"
+    "bootstrap_generator_test"
```

Comment:
We need to figure out the logic on only including this when the branch under test is master. See an example here: https://github.com/grpc/grpc/blob/5b46c8bdbad733fa74ddf07c5b6fa3c9a600dbb9/tools/internal_ci/linux/grpc_xds_k8s_xlang.sh#L59-L62

You kinda need to do the opposite - add the `bootstrap_generator_test` to the list of tests only if `$TESTING_VERSION` is `master.

Reply:
Done.